apps/frontend/src/components/TodayMeetings.tsx

```
@@ -1,173 +1,57 @@
-  if (isLoading) {
+  {
+    /* if (isLoading) {
     return (
       <ol>
         <li className="text-lg font-medium text-[#DCDCDD]/80">
           <SkeletonCard />
         </li>
       </ol>
     )
+  } */
+  }
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)